.nvmrc

```diff
@@ -1 +1 @@
-v18.17.0
+20
```

Comment:
This is the syntax for pinning to a major version and using the latest minor version (essentially `^20.0.0`)